src/app/models/admin_feedback.server.model.ts

```diff
@@ -0,0 +1,59 @@
+import { Mongoose, Schema } from 'mongoose'
+
+import { IAdminFeedbackModel, IAdminFeedbackSchema } from '../../types'
+
+import { USER_SCHEMA_ID } from './user.server.model'
+
+// also acts as the model Id
+export const ADMIN_FEEDBACK_COLLECTION_NAME = 'adminFeedback'
+
+const AdminFeedbackSchema = new Schema<
+  IAdminFeedbackSchema,
+  IAdminFeedbackModel
+>(
+  {
+    userId: {
+      type: Schema.Types.ObjectId,
+      ref: USER_SCHEMA_ID,
+      required: true,
+    },
+    rating: {
```

Comment:
Yeah, I think kinda want to make the field extendible to the future if we want to expand the number rating. Do you think it will make more sense if we just allow rating to be 1-5 and then up -> 5, down -> 1? 

So I set it as 0|1 so that when we display our metric we can show it as a fraction of 'satisfaction', it currently effectively work as a boolean 😅

Reply:
Have discussed with Stacey! it will most likely stick to 'Up' and 'Down' as its the most actionable rating, and simple for quick engagement by the admins too!

Reply:
Throwing my two cents into this debate.

I think `rating: Number` represents well onto what it is currently, and what it might be in the future. Ratings are usually represented as a numerical value (as opposed to binary) in most places. Thus, giving points for extensibility. A data fix, although valid, would come with risks that could have been mitigated.

Usage of `boolean` has its merits of being smaller and safer (no need to handle for overflow etc), but it is unlikely that this will be a concern for us. Within the context of FormSG, we don't have that much traffic to deal with, and shouldn't optimize it too early.

As for the issue of data skew, usually it is understandable that the data would be skewed, and therefore actions to ignore certain subset of the data is done.

As the suggested change doesn't affect critical the flow of FormSG, I suggest that we lean on having a bias for action and accept this minor imperfection to ship this.